hw/xbox/smbus_xbox_smc.c

```diff
@@ -56,7 +56,9 @@
 #define     SMC_REG_AVPACK_SCART        0x00
 #define     SMC_REG_AVPACK_HDTV         0x01
 #define     SMC_REG_AVPACK_VGA_SOG      0x02
+#define     SMC_REG_AVPACK_RFU          0x03
 #define     SMC_REG_AVPACK_SVIDEO       0x04
+#define     SMC_REG_AVPACK_MODE5        0x05
```

Comment:
I wouldn't even bother exposing this for now. I doubt we'll *ever* need it. And if someone required it, they could just add it themselves.

There's no good way to name it as-is and we *know* that some idiot will pick it and complain that it doesn't work :P